site/gatsby-site/cypress.config.js

```diff
@@ -1,6 +1,11 @@
 const { defineConfig } = require('cypress');
 
+require('dotenv').config();
+
+const config = require('./config');
+
 module.exports = defineConfig({
+  projectId: config.cypress.projectId,
```

Comment:
I think that Cypress automatically picks up `CYPRESS_PROJECT_ID` from the env since this worked before without having to do this 🤔